CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/dispatcher/DispatchersProvider.kt

```diff
@@ -0,0 +1,9 @@
+package co.nimblehq.coroutine.dispatcher
+
+import kotlinx.coroutines.CoroutineDispatcher
+
+interface DispatchersProvider {
+    fun getIO(): CoroutineDispatcher
+    fun getMain(): CoroutineDispatcher
+    fun getDefault(): CoroutineDispatcher
+}
```

Comment:
```suggestion
interface DispatchersProvider {

    fun getIO(): CoroutineDispatcher
    
    fun getMain(): CoroutineDispatcher
    
    fun getDefault(): CoroutineDispatcher
}
```

For consistency. 😁  (You can have a look at other interfaces of the project.)

Reply:
@lydiasama I agree with this.